### DISCUSSION TITLE
how to reduce RMS force error

### DISCUSSION THREAD

**Discussion by changxiaoju** [deepmodeling/deepmd-kit#823] [Q&A]
To reduce the force error of my model, I made the following change in the input.json
for model1: 
`"loss": {
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0.02,
        "limit_pref_v": 1
    },
`
for model2:
`"loss" : {
        "start_pref_e": 1,
        "limit_pref_e": 1,
        "start_pref_f": 10000,
        "limit_pref_f": 10000,
        "start_pref_v": 1,
        "limit_pref_v": 1
    },`
that i increase the training weight of force,but it results in a minor derease of force error and the increase of energy error:
![image](https://user-images.githubusercontent.com/44628664/124272835-db2ed900-db71-11eb-8fa5-174f8174f6e4.png)

**Comment by AnguseZhang** [comment]
From 1 to 2, this will definitely cause the large decrease of energy fitting accuracy, since the factor of forces is always 10000 and far larger than the factor of energies. Then energy information is nearly not involved.

**Reply by changxiaoju** [reply]
But the error in energy doesn't increase more than an order of magnitudethe and 0.2(start_pref_e)/1000(start_pref_f) in model 1 is already more than 1(start_pref_e)/10000(start_pref_f) in model 2. what i concerned is how to reduce RMS force error since the parameter i changed in loss did not work

**Reply by njzjz** [reply]
You should pay more attention to `limit_pref_f` as it will affect the final results.

**Reply by changxiaoju** [reply]
ok,thank you
